backend/docker-compose.yml

```
@@ -7,6 +7,7 @@ services:
       - "8080:8080"
     depends_on:
       - mongo
+      - postgres
     environment:
       SPRING_DATA_MONGODB_URI: mongodb://ztbzBackend:example@mongo:27017/ztbzDatabase
```

Env mentioned in previous comment
```suggestion
      SPRING_DATA_MONGODB_URI: mongodb://ztbzBackend:example@mongo:27017/ztbzDatabase
      SPRING_DATASOURCE_URL: jdbc:postgresql://postgres:5432/default_database
      SPRING_DATASOURCE_USERNAME: adminRoot
      SPRING_DATASOURCE_PASSWORD: example
```